### PR TITLE
pforth 28 (new formula)

### PR DESCRIPTION
**Pull request by SeekingMeaning**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
license audit failure which is expected (Homebrew/brew#7874)

**Comment by chenrui333** [comment]
@SeekingMeaning thanks for the PR, can you rebase to the master? We have sorted out the CI issue. Thanks!

File: Formula/pforth.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class Pforth < Formula
+  desc "Portable ANS-like Forth written in ANSI C"
+  homepage "http://www.softsynth.com/pforth/"
+  url "https://github.com/philburk/pforth/archive/d71efe1792a1414a5b102ae8e69f3913c70aa8b0.tar.gz"
```
Comment (chenrui333) [inline Formula/pforth.rb:4]: This is interesting, why does the upstream dont like to make a release tag for the commit?

Comment (SeekingMeaning) [reply Formula/pforth.rb:4]: Hmm should I ask about this in an issue on their repo?

Comment (SMillerDev) [reply Formula/pforth.rb:4]: Yeah, we require a tagged commit to release

Comment (chenrui333) [reply Formula/pforth.rb:4]: Another interesting to call out is the repo does not have OSS license either.

**Comment by chenrui333** [comment]
Going to mark the PR as `do not merge` due to the two concerns that we raised.

**Comment by SMillerDev** [comment]
Since it doesn't match https://docs.brew.sh/Acceptable-Formulae#stable-versions and https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae I'm just closing this.

**Pull request closed without merging**